kura/org.eclipse.kura.rest.provider/src/main/java/org/eclipse/kura/internal/rest/provider/PasswordAuthenticationProvider.java

```diff
@@ -45,14 +44,13 @@
 @Priority(200)
 public class PasswordAuthenticationProvider implements AuthenticationProvider {
 
-    private static final String PASSWORD_AUTH_FAILED_MSG = "{} Rest - Failure - Password Authentication failed";
+    private static final String PASSWORD_AUTH_FAILED_MSG = "{} Rest - Failure - Authentication failed as username or password not matching";
 
```

Comment:
Or simply "Authentication failed"?

Reply:
There is already a message for authentication failed. I believe this message here is used in audit log to report that the user has failed a login using the password provider.
I think this can be helpful, while inspecting the logs, to understand when and where the user or attacker tried to access.

Reply:
We have a similar message in the Certificate auth.